aim/db/infra_model.py

```diff
@@ -164,3 +165,54 @@ class ApicAssignment(model_base.Base, model_base.AttributeMixin):
     aim_aid_id = sa.Column(sa.String(64))
     last_update_timestamp = sa.Column(sa.TIMESTAMP, server_default=func.now(),
                                       onupdate=func.now())
+
+
+class VlanRange(model_base.Base):
+    """DB model for vlan ranges under a NestedParameter."""
+
+    __tablename__ = 'aim_vlan_ranges'
+
+    nested_parameter_cluster_name = sa.Column(
+        sa.String(64), sa.ForeignKey('aim_nested_parameter.cluster_name'),
```

Comment:
Is 64 large enough to support the cluster name?

Reply:
aim_nested_parameter.cluster_name is also a string of length 64 so it should be good.